src/Faker/Provider/hi_IN/Address.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+namespace Faker\Provider\hi_IN;
+
+class Address extends \Faker\Provider\Address
+{
+
+    /**
+    * city and state names from :
+    * @link https://github.com/joke2k/faker/blob/master/faker/providers/address/hi_IN/__init__.py
+    */
+    protected static $city = array(
```

Comment:
please group city names, one line per first letter (see other providers for an example)